apps/chat/src/store/share/share.epics.ts

```diff
@@ -324,7 +397,14 @@ const acceptInvitationSuccessEpic: AppEpic = (action$) =>
   action$.pipe(
     filter(ShareActions.acceptShareInvitationSuccess.match),
     switchMap(({ payload }) => {
-      history.replaceState({}, '', window.location.origin);
+      if (payload.isApplication) {
+        window.location.replace('/marketplace');
+
+        //TODO make request for the shared applications to add them into the state when share invitation is accepted.
+        return of(ModelsActions.getModels());
```

Comment:
"shared with me" application should be added in installed (My workspace)
I'm not sure if need to show Toast-notification or no
```suggestion
        return concat(
          of(ModelsActions.getModels()),
          of(
            ModelsActions.addInstalledModels({
              references: [payload.acceptedId],
              showSuccessToast: false,
            }),
          ),
        );
```

Reply:
As we decided we will implement this during the next iteration.